workspace/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"gopkg.in/go-playground/validator.v9"
+)
+
+type inputUserData struct {
+	FirstName    string `validate:"required"`
+	LastName     string `validate:"required"`
+	Age          int    `validate:"required,gte=18,lte=80"`
+	City         string `validate:"required"`
+	Organization string `validate:"required,contains=/"`
+}
+
+type outputUserData struct {
+	FullName      string
+	Age           int
+	City          string
+	Organization  string
+	Department    string
+	Subdepartment string
+	Team          string
+}
+
+func OpenFile() []byte {
+	inputByte, err := ioutil.ReadFile("input.json")
+
+	if err != nil {
+		fmt.Print(err)
+		os.Exit(1)
+	}
+
+	return inputByte
+}
+
+func CreateInputStruct() inputUserData {
+	inputByte := OpenFile()
+
+	inputStruct := inputUserData{}
+
+	_ = json.Unmarshal([]byte(inputByte), &inputStruct)
+
+	return inputStruct
+}
+
+func ValidateData() {
```

Comment:
This function should has `inputStruct` as an argument. Creating input inside is too much.